runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -1660,19 +1679,107 @@ static void buildInterfaceCall(TR::CodeGenerator *cg, TR::Node *callNode, TR::Re
    generateCompareInstruction(cg, callNode, vftReg, tmpReg, true);
    TR::LabelSymbol *firstBranchAddressCacheSlotLabel = ifcSnippet->getFirstBranchAddressCacheSlotLabel();
 
-   generateTrg1ImmSymInstruction(cg, TR::InstOpCode::ldrx, callNode, tmpReg, 0, firstBranchAddressCacheSlotLabel);
-   generateConditionalBranchInstruction(cg, TR::InstOpCode::b_cond, callNode, hitLabel, TR::CC_EQ);
+   TR::Compilation *comp = cg->comp();
+   TR_Debug * debugObj = cg->getDebug();
```

Comment:
Please remove the space between `*` and `debugObj`.

Reply:
fixed.